tests/functional/pv/pvc_clone/test_pvc_to_pvc_clone.py

```diff
@@ -106,7 +106,9 @@ def test_pvc_to_pvc_clone(self, interface_type, setup, teardown_factory):
             sc_name, parent_pvc, clone_yaml, namespace
         )
         teardown_factory(cloned_pvc_obj)
-        helpers.wait_for_resource_state(cloned_pvc_obj, constants.STATUS_BOUND)
+        helpers.wait_for_resource_state(
+            cloned_pvc_obj, constants.STATUS_BOUND, timeout=120
```

Comment:
could you increase to 300, whenever there is load on cephfs it takes more time to bound

Reply:
Done